### DISCUSSION TITLE
curl usage

### DISCUSSION THREAD

**Discussion by lbdyck** [zopencommunity/meta#159] [General]
It should be noted that the suggested installation command using pax and curl requires the use of the ZOSOpenTools version of curl.

**Comment by MikeFultonDev** [comment]
I didn’t know that. Can you show what happens with another curl and what version?

**Reply by lbdyck** [reply]
This is what happens 

![image](https://user-images.githubusercontent.com/42328411/216789175-c5ee3849-9c2b-48a2-b226-7198fb3bde0c.png)

**Reply by IgorTodorovskiIBM** [reply]
Oh, I think that's a z/OS /bin/sh issue. Good to know

**Reply by MikeFultonDev** [reply]
@IgorTodorovskiIBM did the docs get updated to reflect this? Looks like there is a pre-req to either use `bash` as the shell or use a different pax incantation?

**Reply by IgorTodorovskiIBM** [reply]
The release page one-liner install has been updated to work on /bin/sh

**Comment by MikeFultonDev** [comment]
Closing this off - I've opened an issue here: https://github.com/ZOSOpenTools/meta/issues/230
